class MyException(Exception):
    pass
try:
    open.
except IOError as err:
    raise StevesException("IO error detected; {}".format(err))
